JeffKersting/ideabox
# Feature/final fixes

#### What does  this PR do?

Removes final code extraneous comments. Fixes minor styling issues. Adds README doc with related images.

#### Where should the reviewer start?

Read the README. Review code.

#### How should this be manually tested?

Add multiple idea cards. Favorite idea cards. Comment on idea cards. Refresh page to check data persistence. Delete idea cards. Show favorited. Show comments. Search ideas.
